app/assets/stylesheets/buttons.scss

```diff
@@ -8,14 +8,32 @@
 }
 
 .btn {
+  display: inline-block;
   font-weight: 600;
+  font-size: 14px;
+  padding: 0.75rem 2rem;
+  border-radius: 2rem;
+  text-align: center;
+  border: none;
 
   &-primary {
     background-color: $color-primary-0;
     color: white;
 
     &:hover, &:focus {
       background-color: $color-primary-1;
+      color: white;
+    }
+
+    &-border {
+      background-color: white;
+      box-shadow: inset 0 0 0 2px $color-primary-0;
+      color: $color-primary-0;
+
+      &:hover, &:focus {
+        background-color: rgba(6,71,99,0.25);
```

Comment:
this can be extracted to a variable. You can also use 8 character hex codes.

Reply:
It's a lot easier to tweak the opacity when testing using RGBA, but I'll extract it.

Reply:
I live in hex for colours now